webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -49,16 +49,6 @@ function SystemUsers(props: Props) {
     const [userCount, setUserCount] = useState<number | undefined>();
     const [loadingState, setLoadingState] = useState<LoadingStates>(LoadingStates.Loading);
 
-    // Effect to get the total user count
-    useEffect(() => {
-        const getUserCount = async () => {
-            const {data} = await props.getUserCountForReporting({});
-            setUserCount(data);
-        };
-
-        getUserCount();
-    }, []);
-
     // Effect to get the user reports
     useEffect(() => {
         async function fetchUserReportsWithOptions(tableOptions?: {
```

Comment:
@M-ZubairAhmed Something came to mind while working on this. If we are waiting a relatively long time for the request to come back from this function, what happens if we change the filter in the meantime? Do we have a way to cancel this request, or at the very least disregard it?

Reply:
There is abortController which can be used to cancel fetch https request, however we do not have a system in place in our Client.js to support that yet (could be a good platform task), so we can just utilize the useEffect with correct dependencies which will relies on React to give us the latest total value dependent on the filter change